drools-wb-webapp/pom.xml

```diff
@@ -1240,6 +1240,22 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
```

Comment:
Is this confirmed from CLI (```mvn gwt:run```) and IDEA?

I had to add it as the top-most dependency (or probably just before any other dependency that has ```drools-compiler``` as a transitive dependency). You've added last.